libs/sdk-backend-tiger/tests/integrated/backend.ts

```diff
@@ -66,5 +63,6 @@ export function testBackend(): IAnalyticalBackend {
 
 export function testWorkspace(): string {
     // UI SDK Reference workspace ID
-    return "4dc4e033e611421791adea58d34d958c";
+    const ws = process.env.WORKSPACE_ID;
```

Comment:
How about the integrated test with mock ? please ensure that at least it passed https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13025/